docs/pages/enroll-resources/machine-id/workload-identity/federation.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: SPIFFE Federation
+description: An overview of the Teleport Workload Identity SPIFFE Federation feature.
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. This means that some
+features may be missing. We're actively looking for design partners to help us
+shape the future of Workload Identity and would love to
+[hear your feedback](mailto:product@goteleport.com).
+</Admonition>
+
+Federation is the ability for a trust relationship to be established between
+two trust domains, allowing workloads in one trust domain to validate the
+identities of workloads in the other trust domain.
+
+SPIFFE sets out a standard for federation that allows trust domains managed by
+different implementations to federate with one another. For example, a trust
```

Comment:
I'm not really sure from the introduction of this guide what functionality Teleport makes available. This sounds more like an overview of a SPIFFE concept. The only mention of Teleport is in this sentence, which sounds like a possible use case, but not a feature.

Reply:
I had thought I'd captured this in the first sentence but I'll have another think at if I can make this clearer.

Reply:
[950cc23](https://github.com/gravitational/teleport/pull/46020/commits/950cc23ece2a6ff8b25f31306ba78588bc54a186)